design_docs/analysis_task_template.md

```diff
@@ -0,0 +1,736 @@
+<h2> Title: Analysis Task Template <br>
+Xylar Asay-Davis <br>
+date: 2017/03/08 <br>
+</h2>
+<h3> Summary </h3>
+A new template python file for analysis tasks will be added to the repository.
```

Comment:
If you add a line here it markdown will render the *must* correctly.  For whatever reason having no line in between the header and the text is causing problems with rendering of markdown on github, but this solution (extra blank line) appears to fix the problem.